packages/IngestAPI/src/main/resources/application.yml

```diff
@@ -9,11 +9,17 @@ spring:
       value-deserializer: org.springframework.kafka.support.serializer.ErrorHandlingDeserializer
       properties:
         spring:
-          json.trusted.packages: nz.govt.eop.ingestapi
+          json.trusted.packages: nz.govt.eop.messages
           deserializer.value.delegate.class: org.springframework.kafka.support.serializer.JsonDeserializer
+          json.use.type.headers: false
+          # TODO: Configure this alongside consumers when we add more message types
```

Comment:
This requires more [config-wrangling](https://docs.spring.io/spring-kafka/docs/current/reference/html/#messaging-message-conversion). I figured we could defer since this PR has already taken some time, but happy for pushback of we think it's worth doing now.